subsys/bootloader/bl_crypto/CMakeLists.txt

```diff
@@ -29,3 +29,6 @@ elseif (CONFIG_SB_CRYPTO_CC310_SHA256)
 else()
   message(FATAL_ERROR "No hash implementation chosen for bootloader.")
 endif()
+
+zephyr_include_directories(.)
```

Comment:
This shouldn't be needed anymore.

Reply:
You're right, I'll remove it in a later PR.